### PR TITLE
Add more structure to credit file reference data

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: openapi/components/schemas/KycDocument/CreditFileMatches.yaml [hunk 1]
```diff
@@ -53,6 +53,41 @@ properties:
...
+        description: Credit file create date.
+        type: string
+        readOnly: true
+      numberOfTradesOnFile:
+        description: Number of trades on file.
+        type: string
+        readOnly: true
+        example: 001
+      singleDecision:
+        description: Data related to single source decision.
```
Comment (adamaltman) [inline openapi/components/schemas/KycDocument/CreditFileMatches.yaml:84]: is this nullable too?

**Comment by dara123** [comment]
@Rebilly/php please review

**Comment by dara123** [comment]
@Rebilly/php please review

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/KycDocument/CreditFileCommonDecisionData.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+readOnly: true
+type: object
+properties:
+  lastNameMatch:
+    description: Last name match.
+    type: string
+    readOnly: true
+    example: Y
+  firstNameMatch:
+    description: first name match.
```
Comment (colinbird) [inline openapi/components/schemas/KycDocument/CreditFileCommonDecisionData.yaml:10]:
```suggestion
    description: First name match.
```

**Review by colinbird — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**